parser.py

```diff
@@ -9,7 +10,14 @@ def headlines(design = "APEX"):
 
     # split into physical attributes and simulation results
     rocket = root[0]
-    sims = root[1]
+    sims   = root[1]
+
+    def simdata(rootObj,dataset,att):
```

Comment:
I think you forgot to return the result here

Reply:
done